P1/Project_Code/frontend/src/components/Navigation/NavigationBar.jsx

```diff
@@ -15,13 +15,18 @@ import {
 
 import { UserContext } from "../../context/User.context.jsx";
 import avatar from "../../assets/image-avatar.png";
+import useViewport from "../../services/useViewport.jsx";
 
 export const NavigationBar = () => {
   const location = useLocation();
   const [activePath, setActivePath] = useState(location.pathname);
-
+  const { width } = useViewport();
   const { user, logoutUser } = useContext(UserContext);
 
+  const isMobile = width <= 767;
```

Comment:
https://mui.com/material-ui/customization/breakpoints/
https://mui.com/material-ui/react-use-media-query/
look at how to do screen checks this way

Reply:
Agreed.I'm creating a backlog ticket to refactor this code to use useMediaQuery hook.